### PR TITLE
Css variables PR

### PR DESCRIPTION
**Pull request by wchrapcynski**
Added CSS variables for a default color palette, images, and font weights in App.css all for issue #101 . More will probably be needed but I think this is a good start for everyone to use.

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+    --social-facebook: url("../../assets/images/social_facebook.png");
+    --social-instagram: url("../../assets/images/social_instagram.png");
+    --social-pinterest: url("../../assets/images/social_pinterest.png");
+    --slash-background: url("../../assets/images/splash_background.png");
+    --text-box-search: url("../../assets/images/text_box_search.png");
+    /* Font Weights */
+    --paragraph-weight: 14px;
+    --banner-weight: 80px;
+    --link-weight: 24px;
+    --title-weight: 36px;
```
Comment (ZakkMan) [inline client/src/App.css]: These need to be in units of 100 (i.e. 100, 200, 300, etc up to 900) - like we talked about in person

Comment (wchrapcynski) [reply client/src/App.css]: Okay. This is fixed and should be ready. There's a pull req for it currently.

Comment (wchrapcynski) [reply client/src/App.css]: @ZakkMan Can we get this merged ASAP so that everyone can get to using them? Thanks!

**Comment by ZakkMan** [comment]
@wchrapcynski Great Work!

**Pull request merged**